### DISCUSSION TITLE
Testing Lightning salesforce version cypress is stuck and not moving forward

### DISCUSSION THREAD

**Discussion by FarazQualitest21** [cypress-io/cypress#28753] [Component Testing]
The moment i am logging in salesforce lighting my cypress is stuck (Your tests are loading)
<img width="1512" alt="Screenshot 2024-01-19 at 10 47 00" src="https://github.com/cypress-io/cypress/assets/152856590/494274c0-054d-4583-a02c-5244e887a131">

Do someone have any solution that how can i test saleforce lightning version using cypress

**Comment by mitchwd** [comment]
Wouldn't be surprised if this has to do with the Salesforce login process and iframes. Cypress as a whole doesn't like/support iframes, which makes testing Salesforce apps nearly impossible. https://docs.cypress.io/guides/guides/web-security#Cross-origin-iframes

I had something working 3-4 years ago, but it was very flaky and constantly breaking.
Might be worth looking at another library like [Playwright](https://playwright.dev/) or [UTAM](https://utam.dev/), which has more support for iframes.

Best of luck!
